operations/power_management/Shut_Down_and_Power_Off_Compute_and_User_Access_Nodes.md

```diff
@@ -5,134 +5,156 @@ Shut down and power off compute and user access nodes \(UANs\). This procedure p
+   **Note:** In certain cases, the command may display an error similar to the following:
 
-    The command to run is displayed in the output of the `sat bootsys shutdown` command.
+   ```text
+   ERROR: Failed to get state of nodes in session template 'UAN_SESSION_TEMPLATE': Failed to get state of nodes with role=['Application', 'Application_UAN'] for boot set 'BOOT_SET' of session template 'UAN_SESSION_TEMPLATE': GET request to URL 'https://api-gw-service-nmn.local/apis/smd/hsm/v2/State/Components' failed with status code 400: Bad Request. Bad Request Detail: bad query param: Argument was not a valid HMS Role
+   ```
 
-    ```bash
-    kubectl logs -n services -c boa -f \
-                 --selector job-name=boa-boa-79584ffe-104c-4766-b584-06c5a3a60996
-    ```
+   This error is a non-fatal error and does not affect the `bos-operations` stage of `sat bootsys`.
 
-    Example output:
+   **Note:** In certain cases, the command may fail before reaching the displayed timeout
+   and will show warnings similar to the following:
 
-    ```
-    2020-08-21 17:27:02,358 - DEBUG   - cray.boa - BOA starting
-    2020-08-21 17:27:02,358 - DEBUG   - cray.boa - Boot Agent Image:  created.
-    2020-08-21 17:27:02,358 - INFO    - cray.boa - Boot Session: boa-79584ffe-104c-4766-b584-06c5a3a60996
-    2020-08-21 17:27:02,371 - INFO    - cray.boa.connection - Reattempting GET request for 'http://cray-cfs-api/apis/cfs/sessions'
-    2020-08-21 17:27:02,373 - INFO    - cray.boa.connection - Reattempting GET request for 'http://cray-cfs-api/apis/cfs/sessions'
-    2020-08-21 17:27:02,395 - INFO    - cray.boa.connection - Reattempting GET request for 'http://cray-cfs-api/apis/cfs/sessions'
-    2020-08-21 17:27:02,437 - INFO    - cray.boa.connection - Reattempting GET request for 'http://cray-cfs-api/apis/cfs/sessions'
-    2020-08-21 17:27:02,519 - INFO    - cray.boa.connection - Reattempting GET request for 'http://cray-cfs-api/apis/cfs/sessions'
-    2020-08-21 17:27:02,681 - INFO    - cray.boa.connection - Reattempting GET request for 'http://cray-cfs-api/apis/cfs/sessions'
-    2020-08-21 17:27:03,003 - INFO    - cray.boa.connection - Reattempting GET request for 'http://cray-cfs-api/apis/cfs/sessions'
-    2020-08-21 17:27:03,645 - INFO    - cray.boa.connection - Reattempting GET request for 'http://cray-cfs-api/apis/cfs/sessions'
+   ```text
+   WARNING: The 'kubectl wait' command failed instead of timing out. stderr: error: condition not met for jobs/boa-79584ffe-104c-4766- b584-06c5a3a60996
+   ```
 
-    ```
+   The existence of these warnings does not prevent the BOS operation from proceeding, however it
+   may result in the `bos-operations` stage of the `sat bootsys` command exiting before the BOS
+   operation is complete. In this case it is especially important to view the logs to monitor the
+   shutdown, and to verify that the nodes reached the expected power state, both of which are shown
+   in the remaining steps.
```

Comment:
```suggestion
    The BOS operation can still proceed even with these warnings. However, the warnings
    may result in the `bos-operations` stage of the `sat bootsys` command exiting before the BOS
    operation is complete. Because of this, it is important to view the logs in order to monitor the
    boot and to verify that the nodes reached the expected state. Both of these recommendations are shown
    in the remaining steps.
```